dali/test/python/test_pipeline.py

```diff
@@ -1656,42 +1701,42 @@ def test_preserve_arg():
     pipe = dali.Pipeline(1, 1, 0)
     with pipe:
         out = dali.fn.external_source(source=[[np.array([-0.5, 1.25])]], preserve=True)
-        res = dali.fn.resize(out, preserve=True)
+        res = dali.fn.resize(out, preserve=True)  # noqa: F841
```

Comment:
out of curiosity. What's the problem here?

Reply:
Oh, maybe it is silly: the test checks preservation of unused operators if preserve is set to True. So the linter complains about unused variable. I could very well just call resize and not assign the result to any variable, but wanted to preserve this more "canonical" look of defining the pipeline, after all the variable is unused intentionally.